content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -5,6 +5,7 @@ reviewers:
 - liggitt
 content_template: templates/task
 weight: 30
+min-kubernetes-server-version: v1.16
```

Comment:
this doc applies to earlier versions (we include both v1 and v1beta1 examples)

Reply:
OK, that makes sense. Thanks.